query/language/define.feature

```diff
@@ -937,17 +1032,23 @@ Feature: TypeQL Define Query
       | datetime   | graduation-date   |
 
 
-  # TODO
-#  Scenario Outline: a type can own a '<value_type>' attribute type as a key
-#
-#  Scenario Outline: a '<value_type>' attribute type is not allowed to be a key
+  Scenario: defining an attribute type value when a different value is already defined errors
+    Then typeql schema query; fails with a message containing: "different 'name value string' is already defined"
+      """
+      define name value long;
+      """
+
+  ###########
+  # STRUCTS #
+  ###########
 
+    # TODO 3.x: Add tests for structs
 
   ###############
   # ANNOTATIONS #
   ###############
 
-  # TODO: Add tests for structs and their fields
```

Comment:
It's my todo, I consider that "Add tests for structs" is enough to force the developer thinking about all tests for structs.